CMakeLists.txt

```diff
@@ -27,6 +27,9 @@ include(cmake/dd-macros.cmake)
 include(cmake/dd-testing.cmake)
 include(cmake/dd-version.cmake)
 
+use_cpp11()
+ddqt()
```

Comment:
This would benefit from a comment; if the definition wasn't conveniently visible in the diff a few lines below, I would have no idea what it does, and the name is in no way helpful.